src/module.c

```diff
@@ -304,6 +305,17 @@ void ReplyWithSeriesLabels(RedisModuleCtx *ctx, const Series *series) {
     }
 }
 
+void ReplyWithSeriesLastDatapoint(RedisModuleCtx *ctx, const Series *series) {
+    if(SeriesGetNumSamples(series)==0){
+        RedisModule_ReplyWithArray(ctx, 0);
```

Comment:
Please cover in test

Reply:
@ashtul please revise latest commit with extra testing on `mget` with series with empty data.